### PR TITLE
Install Wizard Templates - Added Sidecars/Images

### PR DESCRIPTION
**Pull request by ChristianAtDell**
# Description
Per the listed Issue, templates in the install wizard were not providing sidecars in some cases. This PR is an effort to go and add all appropriate sidecars to relevant templates in the Wizard, as well as update images in containers therein to be more accurate. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1470 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Comment by anandrajak1** [comment]
Please update sidecars from 1.12.0 templates

**Review by anandrajak1 — CHANGES_REQUESTED** [review]

File: content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template [hunk 1]
```diff
@@ -39,7 +39,7 @@ spec:
     forceRemoveDriver: true
     common:
       # Image for CSI PowerScale driver v2.12.0
-      image: "dellemc/csi-isilon:v2.12.0"
+      image: "quay.io/dell/container-storage-modules/csi-isilon:v2.12.0"
```
[the remark this answers is not in the document]
Comment (ChristianAtDell) [reply content/docs/deployment/csminstallationwizard/src/templates/operator/csm-isilon-1.12.0.template:42]: I do not understand the request; Operator sample files for 1.12.0 point to the quay image here, so I am matching that.

**Comment by ChristianAtDell** [comment]
> Please update sidecars only for 1.12.0 templates, please don't update for 1.11.0 and 1.10.1 etc

Why? I copied the sidecars from the matching Operator sample files for those versions. We claim n-2 support, so if the sidecars should be included in n-1 and n-2 templates, we should have them here.

**Review by anandrajak1 — APPROVED** [review]

**Review by mjsdell — APPROVED** [review]
lgtm

**Pull request merged**